requirements/requirements_lightning.txt

```diff
@@ -1,4 +1,5 @@
 hydra-core>1.3,<=1.3.2
+neptune
```

Comment:
Please remove this dependency. All loggers are meant to be optional apart from wandb since it increases NeMos already large dependency list.

Reply:
Hi @titu1994 , can I create a separate requirements_neptune.txt and include it as an optional install extra?

Reply:
No need for that. Let's add an error message if trying to use Neptune without explicitly installing it. Adding a requirements file for every single library is not scalable

Reply:
Removed `neptune` from requirements file.